src/js/Controllers/UIController.js

```diff
@@ -69,6 +69,7 @@ class UIController {
     handleRPC(rpc) {
         let methodName = rpc.method.split(".")[1]
         var appUIState = rpc.params && rpc.params.appID ? store.getState()['ui'][rpc.params.appID] : null;
+        const GENERIC_ERROR = 22;
```

Comment:
@LiliiaShlikhtLuxoft In commit 9d7cecbe159ce12281d52076ba7c858d8f18f50a you've moved `const` , not a `variable`. But commit message is `Fixed location of GENERIC_ERROR variable`. Looks like commit message should be edited